src/templates/haml/home5/index.html.haml

```diff
@@ -0,0 +1,450 @@
+- stash(
+-   title => l('Online Trading platform for binary options on Forex, Indices, Commodities and Smart Indices'),
+-   next_link => '/why-us',
+-   createacc => l('Create free account now'),
+- );
+- my $lang = stash->{language};
+%script{:src=>'//use.typekit.net/bfu3ywb.js'}
```

Comment:
we would need to remove this when we release as we already have typekit loaded on binary.com